diff -up MuseScore-4.4.3/src/app/main.cpp.1~ MuseScore-4.4.3/src/app/main.cpp
--- MuseScore-4.4.3/src/app/main.cpp.1~	2024-11-01 01:51:11.135043799 +0100
+++ MuseScore-4.4.3/src/app/main.cpp	2024-11-01 01:51:53.755582399 +0100
@@ -93,12 +93,6 @@ int main(int argc, char** argv)
     qputenv("QT_STYLE_OVERRIDE", "Fusion");
     qputenv("QML_DISABLE_DISK_CACHE", "true");
 
-#ifdef Q_OS_LINUX
-    if (qEnvironmentVariable("QT_QPA_PLATFORM") != "offscreen") {
-        qputenv("QT_QPA_PLATFORMTHEME", "gtk3");
-    }
-#endif
-
 #ifdef Q_OS_WIN
     // NOTE: There are some problems with rendering the application window on some integrated graphics processors
     //       see https://github.com/musescore/MuseScore/issues/8270
