diff --git a/src/app/main.cpp b/src/app/main.cpp
index a41b69b..a1255c2 100644
--- a/src/app/main.cpp
+++ b/src/app/main.cpp
@@ -106,10 +106,6 @@ int main(int argc, char** argv)
 #endif
 
 #ifdef Q_OS_LINUX
-    if (qEnvironmentVariable("MU_QT_QPA_PLATFORM") != "offscreen") {
-        qputenv("QT_QPA_PLATFORMTHEME", "gtk3");
-    }
-
     //! NOTE Forced X11, with Wayland there are a number of problems now
     if (qEnvironmentVariable("MU_QT_QPA_PLATFORM") == "") {
         qputenv("QT_QPA_PLATFORM", "xcb");
