diff -up MuseScore-3.6.2/bww2mxml/mxmlwriter.cpp.omv~ MuseScore-3.6.2/bww2mxml/mxmlwriter.cpp
--- MuseScore-3.6.2/bww2mxml/mxmlwriter.cpp.omv~	2021-07-20 04:24:54.866299994 +0200
+++ MuseScore-3.6.2/bww2mxml/mxmlwriter.cpp	2021-07-20 04:25:01.789356082 +0200
@@ -25,6 +25,8 @@
 
 #include <QtCore/QtDebug>
 
+using Qt::endl;
+
 #include "mxmlwriter.h"
 
 namespace Bww {
diff -up MuseScore-3.6.2/libmscore/xmlwriter.cpp.omv~ MuseScore-3.6.2/libmscore/xmlwriter.cpp
--- MuseScore-3.6.2/libmscore/xmlwriter.cpp.omv~	2021-07-20 03:33:33.064233007 +0200
+++ MuseScore-3.6.2/libmscore/xmlwriter.cpp	2021-07-20 03:36:11.726434231 +0200
@@ -14,6 +14,9 @@
 #include "property.h"
 #include "scoreElement.h"
 
+using Qt::endl;
+using Qt::dec;
+
 namespace Ms {
 
 //---------------------------------------------------------
diff -up MuseScore-3.6.2/mscore/musescore.cpp.omv~ MuseScore-3.6.2/mscore/musescore.cpp
--- MuseScore-3.6.2/mscore/musescore.cpp.omv~	2021-07-20 03:50:37.361178611 +0200
+++ MuseScore-3.6.2/mscore/musescore.cpp	2021-07-20 03:50:45.665706132 +0200
@@ -153,6 +153,8 @@
 #include "macos/cocoabridge.h"
 #endif
 
+using Qt::endl;
+
 #ifdef AEOLUS
 extern Ms::Synthesizer* createAeolus();
 #endif
diff -up MuseScore-3.6.2/mscore/script/script.cpp.omv~ MuseScore-3.6.2/mscore/script/script.cpp
--- MuseScore-3.6.2/mscore/script/script.cpp.omv~	2021-07-20 03:54:53.645518837 +0200
+++ MuseScore-3.6.2/mscore/script/script.cpp	2021-07-20 03:54:59.456956461 +0200
@@ -19,6 +19,8 @@
 
 #include "libmscore/score.h"
 
+using Qt::endl;
+
 namespace Ms {
 
 //---------------------------------------------------------
diff -up MuseScore-3.6.2/mscore/script/testscript.cpp.omv~ MuseScore-3.6.2/mscore/script/testscript.cpp
--- MuseScore-3.6.2/mscore/script/testscript.cpp.omv~	2021-07-20 03:55:18.640343235 +0200
+++ MuseScore-3.6.2/mscore/script/testscript.cpp	2021-07-20 03:55:27.718969939 +0200
@@ -17,6 +17,8 @@
 
 #include "libmscore/scorediff.h"
 
+using Qt::endl;
+
 namespace Ms {
 
 //---------------------------------------------------------
diff -up MuseScore-3.6.2/mscore/svggenerator.cpp.omv~ MuseScore-3.6.2/mscore/svggenerator.cpp
--- MuseScore-3.6.2/mscore/svggenerator.cpp.omv~	2021-07-20 03:54:25.552280176 +0200
+++ MuseScore-3.6.2/mscore/svggenerator.cpp	2021-07-20 03:54:36.945213390 +0200
@@ -45,6 +45,8 @@
 #include "libmscore/imageStore.h"
 #include "libmscore/mscore.h"
 
+using Qt::endl;
+
 ///////////////////////////////////////////////////////////////////////////////
 // FOR GRADIENT FUNCTIONALITY THAT IS NOT IMPLEMENTED (YET):
 //
