diff -up musescore-3.0.2/mscore/network/loginmanager.cpp.omv~ musescore-3.0.2/mscore/network/loginmanager.cpp
--- musescore-3.0.2/mscore/network/loginmanager.cpp.omv~	2019-02-23 00:53:36.244151000 +0100
+++ musescore-3.0.2/mscore/network/loginmanager.cpp	2019-02-23 00:53:44.284278983 +0100
@@ -571,7 +571,7 @@ void LoginManager::onGetMediaUrlReply(QN
                   }
             }
       else // TODO: handle request error properly
-            qWarning(getErrorString(reply, response).toUtf8().constData());
+            qWarning("%s", getErrorString(reply, response).toUtf8().constData());
 #if 0
       disconnect(_oauthManager, SIGNAL(requestReady(QByteArray)),
             this, SLOT(onGetMediaUrlRequestReady(QByteArray)));
