--- MuseScore-4.2.1/src/importexport/videoexport/internal/videoencoder.cpp.orig	2024-04-10 10:27:03.144740115 +0100
+++ MuseScore-4.2.1/src/importexport/videoexport/internal/videoencoder.cpp	2024-04-10 10:27:13.079384105 +0100
@@ -352,7 +352,7 @@
 
     convertImage_sws(img);
 
-    m_ffmpeg->ppicture->pts = av_rescale_q(m_ffmpeg->codecCtx->frame_number, m_ffmpeg->codecCtx->time_base,
+    m_ffmpeg->ppicture->pts = av_rescale_q(m_ffmpeg->codecCtx->frame_num, m_ffmpeg->codecCtx->time_base,
                                            m_ffmpeg->videoStream->time_base);
 
     int ret = avcodec_send_frame(m_ffmpeg->codecCtx, m_ffmpeg->ppicture);
