diff -up MuseScore-3.0.4/CMakeLists.txt.1~ MuseScore-3.0.4/CMakeLists.txt
--- MuseScore-3.0.4/CMakeLists.txt.1~	2019-02-28 12:31:10.000000000 +0100
+++ MuseScore-3.0.4/CMakeLists.txt	2019-03-06 19:07:29.016202320 +0100
@@ -786,7 +786,7 @@ if (ZERBERUS)
 endif (ZERBERUS)
 
 if (OMR)
-      subdirs (omr thirdparty/poppler)
+      subdirs (omr)
 endif (OMR)
 
 if (OSC)
@@ -829,8 +829,7 @@ else (USE_SYSTEM_FREETYPE)
 endif (USE_SYSTEM_FREETYPE)
 
 if (OMR)
-   include_directories(${PROJECT_SOURCE_DIR}/thirdparty/poppler)
-   include_directories(${PROJECT_SOURCE_DIR}/thirdparty/poppler/poppler)
+   include_directories(/usr/include/poppler)
 endif (OMR)
 
 add_subdirectory(mscore)
diff -up MuseScore-3.0.4/mscore/CMakeLists.txt.1~ MuseScore-3.0.4/mscore/CMakeLists.txt
--- MuseScore-3.0.4/mscore/CMakeLists.txt.1~	2019-03-06 19:06:20.513464039 +0100
+++ MuseScore-3.0.4/mscore/CMakeLists.txt	2019-03-06 19:07:48.926416917 +0100
@@ -501,6 +501,8 @@ target_link_libraries(mscore
       qzip
       beatroot
       kqoauth
+      poppler
+      poppler-qt5
       )
 
 if (BUILD_CRASH_REPORTER)
