--- MuseScore-2.1/CMakeLists.txt.omv~	2017-05-13 16:35:30.399946688 +0200
+++ MuseScore-2.1/CMakeLists.txt	2017-05-13 16:36:38.026666561 +0200
@@ -586,6 +586,8 @@ precompiled_header(QT_INCLUDES all ${BUI
 ADD_CUSTOM_TARGET(mops1 DEPENDS ${PROJECT_BINARY_DIR}/all.h)
 ADD_CUSTOM_TARGET(mops2 DEPENDS ${PCH})
 
+add_definitions(-include ${PROJECT_BINARY_DIR}/all.h)
+
 ##
 ## add subdirs
 ##
--- MuseScore-2.1/all.h.omv~	2017-05-13 16:45:31.432303776 +0200
+++ MuseScore-2.1/all.h	2017-05-13 16:45:54.948551195 +0200
@@ -23,12 +23,14 @@
 
 #include <stdio.h>
 #include <limits.h>
-#include <map>
-#include <set>
 #include <errno.h>
 #include <fcntl.h>
 #include <unistd.h>
 #include <math.h>
+
+#ifdef __cplusplus
+#include <map>
+#include <set>
 #include <functional>
 
 #include <QtGui>
@@ -175,4 +177,4 @@
 #endif
 
 #endif
-
+#endif
